module_utils/network/vyos/utils/utils.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# utils
+
+
+def merge_two_dicts(dict1, dict2):
+    dict = dict1.copy()   # start with dict1's keys and values
+    dict.update(dict2)    # modifies dict with dict2's keys and values & returns None
+    return dict
+
+
+def search_obj_in_list(name, lst):
```

Comment:
If the utils function can be used across platforms this code can be moved in `ansible/module_utils/network/common/`